barretenberg/cpp/src/barretenberg/vm/tests/avm_execution.test.cpp

```diff
@@ -956,26 +956,31 @@ TEST_F(AvmExecutionTests, poseidon2PermutationOpCode)
     std::string bytecode_hex = to_hex(OpCode::CALLDATACOPY) + // opcode CALL DATA COPY
                                "00"                           // Indirect Flag
                                "00000000"                     // cd_offset
-                               "00000004"                     // copy_size
+                               "00000002"                     // copy_size
```

Comment:
Would be good to have a bit more test vectors and especially with the 4 input field elements being different. We would not even catch a regression involving a swap of two input fields.
Is there any reference to "official" test vectors for BN_254 scalar field somewhere?